plugins/README.md

```diff
@@ -0,0 +1,28 @@
+## :plugins
+
+This folder holds all our Gradle- and Kotlin Compiler plugins:
+
+### [:plugins:dataframe-gradle-plugin](./dataframe-gradle-plugin)
+The Gradle plugin for DataFrame that can generate data schemas from a data sample using the simple
+task `dataframes {}`. It uses [:plugins:symbol-processor](./symbol-processor) to generate column accessors.
+
+### [:plugins:symbol-processor](./symbol-processor)
+The KSP plugin that can generate data schemas from a data sample using the `@file:ImportDataSchema` annotation.
```

Comment:
suggestion:
This is an alternative to `dataframes { }` syntax from Gradle, it does the same thing but declared closer to your source code

Reply:
sure, but I'll put it in the symbol-processor/README.md. This is just a summary after all